test/unit/helpers/rubygems_helper_test.rb

```diff
@@ -195,4 +195,34 @@ class RubygemsHelperTest < ActionView::TestCase
       end
     end
   end
+
+  context "change_diff_link" do
+    context "with yanked version" do
+      setup do
+        @version = create(:version, indexed: false)
+        @rubygem = @version.rubygem
+      end
+
+      should "return nil" do
+        assert_nil change_diff_link(@rubygem, @version)
+      end
+    end
+
+    context "with available version" do
+      setup do
+        @virtual_path = "rubygems.aside"
```

Comment:
I think it is easier to provide whole full path in helper for translation. That shortcut is (at least for me) useful in controller/views. But here it is static.